### PR TITLE
Fixed the bug of allowing different EMAIL and NUSNET ID

### PR DESCRIPTION
**Pull request by Siddharth-Sid**
Updated Add and Edit Command functionalities to ensure that the EMAIL and NUSNET ID have the same ID

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ Format: `edit INDEX [n/NAME] [e/Email] [s/STUDENT_ID] [N/NUSNet_ID] [g/GITHUB_ID
     specifying any tags after it.
 
 Examples:
-* `edit 1 p/91234567 e/e0123456@u.nus.edu` Edits the phone number and email address of the 1st person to be `91234567` and `e0123456@u.nus.edu` respectively.
+* `edit 1 p/91234567` Edits the phone number of the 1st person to be `91234567`.
 * `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
 * `edit 1 n/Siddharth Srivastava t/IncompleteIP` Edits the name of the 1st person to be `Siddharth Srivastava` and replaces all existing tags with the tag `IncompleteIP`.
```
Comment (meixuanjin) [inline docs/UserGuide.md:140]: Can consider adding an example showing that changing the NUSnet ID will also change the email

**Review by meixuanjin — CHANGES_REQUESTED** [review]
Line 307 of UG should be edited to reflect the changes in the command summary; other than that LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (Siddharth-Sid) [reply docs/UserGuide.md:141]: Requisite changes made

**Review by meixuanjin — APPROVED** [review]
LGTM

**Pull request merged**